tests/integration/suite/daprd/scheduler/appcallback/http.go

```diff
@@ -110,7 +111,7 @@ func (h *httpcallback) receiveJob(t *testing.T, ctx context.Context, client runt
 	case job := <-h.jobChan:
 		assert.NotNil(t, job)
 		assert.NotNil(t, job.GetData())
-		assert.Equal(t, "/dapr/receiveJobs/test", job.GetMethod())
```

Comment:
I think we want `/dapr/` prefixed as a 'namespace' per say such that users might have a job endpoint and we dont want a conflict

Reply:
This isn't what we do for out other app call back APIs AFAIK though, so would be inconsistent if we did./

Reply:
I think I saw it for `config` & `subscribe`:
```
	router.HandleFunc("/dapr/config", configHandler).Methods("GET")
...
	http.HandleFunc("/dapr/subscribe", subscribeHandler)

```